app/controllers/organizations_controller.rb

```diff
@@ -95,39 +92,10 @@ def organization_params
   end
 
   def set_organization
-    @organization = Organization.find_by(id: organization_params[:id])
+    @organization = @form&.organization || Organization.find_by(id: organization_params[:id])
```

Comment:
when is `@form` or `@form.organization` be `nil`? My question is, when is the second part of the `||` going to be executed?

Reply:
I think that when `generate_new_secret` is called. but I could change that method to use form object too.